briefcase/app.py

```diff
@@ -146,7 +146,7 @@ def find_support_pkg(self):
         s3 = boto3.client('s3', region_name=S3_REGION)
         s3.meta.events.register('choose-signer.s3.*', disable_signing)
 
-        candidates = []
+        top_build_number, top_build = 0, None
```

Comment:
Stylistically, there's no reason for this to be a multi-assignment; could you split this into 2 statements?

Reply:
Sure.